src/cpp/session/modules/clang/CodeCompletion.cpp

```diff
@@ -309,9 +341,13 @@ void discoverSystemIncludePaths(std::vector<std::string>* pIncludePaths)
       else
          includePathOutput = string_utils::trimWhitespace(result.stdOut);
    }
+   
    if (includePathOutput.empty())
       return;
    
+   // check for Apple clang
```

Comment:
Should this be wrapped in `#ifdef __APPLE__` and possibly moved to the location later on where it is used (in the existing __APPLE__)?